src/generator/css/base.less

```diff
@@ -228,21 +260,50 @@ body {
     }
 
     hr {
-        @apply w-4/6 my-8 mx-auto;
+        @apply w-full sm:w-4/6 my-8 mx-auto;
+
+        @media (max-width: 640px) {
+            height: 5px;
+            background-color: #000;
+            margin-left: 20px;
+            width: 90%;
+            margin-bottom: 0.5rem;
+            margin-top: 0.5rem;
+        }
+        @media (min-width: 640px) {
+        display: none;
+        }
     }
 
     .form-container {
-        @apply clear-left pt-10 border border-white pb-2 mb-4 md:w-2/4;
+        @apply clear-left pt-10 border border-[var(--color-body-text)] pb-2 mb-4 md:w-2/4;
+    }
+    @media (max-width: 350px) {
+        .g-recaptcha {
+            transform:scale(0.65);transform-origin:0 0
+        }
     }
 }
 
 .form-wrap {
     position: relative;
     overflow: hidden;
 
-    input,
+    input {
+        @apply lg:w-96 w-72;
```

Comment:
Consider consolidating the width logic into a single tailwind statement. for example:

```
input {
  @apply w-10/12 sm:w-72 lg:w-96;
  min-width: 12rem;
}
```